postfix/edition15.html

```diff
@@ -60,13 +60,14 @@ <h3>
 <h6><a href="#editions">15th edition</a></h6>
 <h6>
 	Author <a href="#author">Ivar Abrahamsen</a>
+	Author <a href="#author">Daniel Hansson</a>
 </h6>
 <h6>
 	License:
 	<a href="http://flurdy.com/docs/license/respect">Respect</a> (CC by-sa)
 <!-- 	on top of <a href="http://creativecommons.org/licenses/by-sa/2.5/">Creative Commons by-sa</a>. -->
 </h6>
-<h6>Last Update: 2020-05-08</h6>
+<h6>Last Update: 2020-12-04</h6>
```

Comment:
Ill revert a local commit I had to do this. Thanks :)

Reply:
I don't understand? Can't you not just merge your local branch?

Reply:
Ah it wasn't needed anymore. All it had was the date update.